pkg/policy/prefilter.go

```diff
@@ -0,0 +1,303 @@
+// Copyright 2016-2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"os/exec"
+	"path"
+	"sync"
+	"syscall"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/maps/cidrmap"
+)
+
+type preFilterMapType int
+
+const (
+	prefixesV4Dyn preFilterMapType = iota
+	prefixesV4Fix
+	prefixesV6Dyn
+	prefixesV6Fix
+	mapCount
+)
+
+const (
+	maxLKeys = 1024 * 64
+	maxHKeys = 1024 * 1024 * 20
```

Comment:
Does the 60 and 20 have any meaning? If yes, can you add a comment with that?

Reply:
No, but I've added a comment anyway.